### ISSUE TITLE
Poor contrast in Find and Plugins Admin windows

### ISSUE THREAD

**Issue by jabcreations** [notepad-plus-plus/notepad-plus-plus#13202]
There is poor contrast on anchors in the Find and Plugins Admin windows. I am using:

- Notepad++ 8.4.8.
- Notepad++ Obsidian theme.
- WindowBlinds program.
- A mildly modified copy of the Fl8t theme (happy to share it with others).

Here is a screenshot:
[Notepad++ Poor Contrast Find Window screenshot](https://community.notepad-plus-plus.org/assets/uploads/files/1677163043757-find-count-bug.png)

It would be nice if anchors were assigned a style variable as I use the instance count in the Find widow _a lot_. If there is an option that I didn't find in my ~10 minutes of looking through the styles please forgive me and let me know where I can find it.

**Comment by alankilborn** [comment]
Somewhat related maybe:  #5087 #5291

**Comment by jabcreations** [comment]
@alankilborn Yes. Seems (at least offhand) likely to be a fairly easy fix one way or another.

**Comment by alankilborn** [comment]
>Seems (at least offhand) likely to be a fairly easy fix one way or another.

Notepad++ accepts user-submitted code changes.
See https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md

**Comment by jabcreations** [comment]
@alankilborn That is fantastic, I'll probably pay one of my future employees to take care of it. For now I'm the sole developer of an entire platform.

**Comment by jabcreations** [comment]
@akittino No worries. I'm simply saying if I can't make the contributions directly myself that I'll manage to get them implemented indirectly. Also while life isn't all that pleasant right now things will be changing for the better very soon and I'm looking forward to contributing to many projects that have helped me over the years.
